src/bases/local/ndlocal.jl

```diff
@@ -56,14 +55,36 @@ function restrict(ϕ::NDRefSpace{T}, dom1, dom2) where T
 
         u = carttobary(dom1, c)
         x = neighborhood(dom1, u)
-
+        σ = sign(dot(normalize(tangents(dom2,1)×tangents(dom2,2)),normal(dom2)))
```

Comment:
We should probably add a function `orientation` for charts that returns +/-1 so that this potentially expensive and definitely ugly line can go.

Reply:
totaly agree